script/DeployPosm.s.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.20;
+
+import "forge-std/console2.sol";
+import "forge-std/Script.sol";
+
+import {IPoolManager} from "@uniswap/v4-core/src/interfaces/IPoolManager.sol";
+import {StateView} from "../src/lens/StateView.sol";
+import {PositionManager} from "../src/PositionManager.sol";
+import {IAllowanceTransfer} from "permit2/src/interfaces/IAllowanceTransfer.sol";
+
+contract DeployPosmTest is Script {
+    function setUp() public {}
+
+    function run(address poolManager, address permit2) public returns (PositionManager posm) {
+        vm.startBroadcast();
+
+        // TODO: remove salt when we have a deployed version on sepolia
```

Comment:
dm-ed you for your thoughts about todos

Reply:
https://github.com/Uniswap/v4-periphery/issues/297